robot/web/web_c.go

```diff
@@ -127,7 +134,8 @@ func (svc *webService) addNewStreams(ctx context.Context) error {
 	}
 
 	for name, source := range svc.videoSources {
-		stream, alreadyRegistered, err := newStream(name)
+		const isVideo = true
+		stream, alreadyRegistered, err := newStream(name, isVideo)
```

Comment:
This `isVideo` parameter/pattern was copied from how the [startup code works](https://github.com/viamrobotics/rdk/pull/3767/files#diff-db223bcebe63e661c7c3c5b7e0cd20523f08be884e758c78aed2d19ec878f5faR177). I just kept it that way to make these different code paths look as similar as possible.